editor/components/post-taxonomies/flat-term-selector.js

```diff
@@ -202,7 +207,10 @@ export default compose(
 		};
 	} ),
 	withSelect( ( select, ownProps ) => {
+		const { getCurrentPost } = select( 'core/editor' );
 		return {
+			hasCreateAction: get( getCurrentPost(), [ '_links', 'wp:action-create-' + ownProps.restBase ], false ),
```

Comment:
Should this logic be its own selector? Something like `canUserForPost( actionType, restBase )` ?

Minor: `getCurrentPost` as implemented is trivial, though nothing guarantees it is, in which case assigning a single variable of `const currentPost = getCurrentPost()` to use across the two merged props would be marginally more performant.

Reply:
> Should this logic be its own selector? Something like `canUserForPost( actionType, restBase )` ?

See #6655